src/indra_cogex/sources/indra_db/__init__.py

```diff
@@ -358,6 +371,9 @@ def get_nodes(self) -> Iterable[Node]:
                                 evidence["pmid"],
                                 ["Publication"],
                             )
+
+                    source_api = evidence["source_api"]
+                    source_api = internal_source_mappings.get(source_api, source_api)
```

Comment:
@cthoyt do you remember if/why this mapping was necessary? I would normally not do this here but I might be missing something.

Reply:
@kkaris is the master of the renaming, this was his suggestion to make sure that the names were consistent

Reply:
I looked into it a bit more deeply and with the latest approach to getting this content, this mapping shouldn't be necessary.